### PR TITLE
1 nav bar

### PR DESCRIPTION
**Pull request by Anshu71**
HavBar and Hero Section done....

### CODE REVIEW

File: src/components/Hero.astro [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+<div>
+  <!-- <hr><hr> -->
+    <section class="text-gray-600 body-font bg-[#101828]">
+        <div class="w-11/12 mx-auto flex md:py-16 py-5 md:flex-row flex-col  items-center">
+        
+          <div class="absolute z-30 py-0 left-0 w-80 h-96  bg-gradient-to-r from-teal-100 via-indigo-500 to-pink-100 opacity-30 blur-3xl flex-none order-4 rounded-xl transform -rotate-45"></div>
+
+          <div class="lg:flex-grow md:w-1/2 lg:pr-0 md:pr-12 flex flex-col md:items-start md:text-left mb-10 md:mb-0 items-center text-center font-title">
+            <h1 class="2xl:text-7xl xl:text-5xl sm:text-4xl text-3xl mb-2 text-white">Customise your computing experience with TCET Linux
```
Comment (JayeshVP24) [inline src/components/Hero.astro]: - [x] make 2xl:text-6xl

File: src/components/Hero.astro [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            </h1>
+            <p class="mb-8 leading-relaxed text-indigo-300 sm:text-sm text-xs">TCET Linux • Powered by FOSS</p>
+            <p class="mb-8 leading-relaxed text-[#D0D5DD] sm:text-base text-sm">TCET Open Source is an organization where software is not just a commodity to be bought and sold, but a collaborative effort that brings people together to solve problems and build something greater than themselves.</p>
+            <div class="flex justify-center">
+              <button class="inline-flex text-white bg-[#01105F] shadow-base shadow-sky-500 border-sky-500 border-y-2 py-2 px-6 focus:outline-none rounded-3xl text:base md:text-lg">Explore</button>
+            </div>
+          </div> 
+
+          <div class=" flex lg:max-w-lg lg:w-full md:w-1/2 w-5/6">
```
Comment (JayeshVP24) [inline src/components/Hero.astro]: - [ ] remove lg:max-w-lg

basically give 50-50 width to main text div and image div

File: src/components/Hero.astro [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+import InstallationSteps from "../components/InstallationSteps";
+import Layout from "../layouts/Layout.astro";
+---
+
+<div>
+  <!-- <hr><hr> -->
+    <section class="text-gray-600 body-font bg-[#101828]">
+        <div class="w-11/12 mx-auto flex md:py-16 py-5 md:flex-row flex-col  items-center">
```
Comment (JayeshVP24) [inline src/components/Hero.astro]: - [x] add little padding bottom

File: src/components/Hero.astro [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        
+          <div class="absolute z-30 py-0 left-0 w-80 h-96  bg-gradient-to-r from-teal-100 via-indigo-500 to-pink-100 opacity-30 blur-3xl flex-none order-4 rounded-xl transform -rotate-45"></div>
```
Comment (JayeshVP24) [inline src/components/Hero.astro]: - [x] add `pointer-events-none` in class names

File: src/components/Hero.astro [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+            </h1>
+            <p class="mb-8 leading-relaxed text-indigo-300 sm:text-sm text-xs">TCET Linux • Powered by FOSS</p>
+            <p class="mb-8 leading-relaxed text-[#D0D5DD] sm:text-base text-sm">TCET Open Source is an organization where software is not just a commodity to be bought and sold, but a collaborative effort that brings people together to solve problems and build something greater than themselves.</p>
+            <div class="flex justify-center">
+              <button class="inline-flex text-white bg-[#01105F] shadow-base shadow-sky-500 border-sky-500 border-y-2 py-2 px-6 focus:outline-none rounded-3xl text:base md:text-lg">Explore</button>
+            </div>
+          </div> 
+
+          <div class=" flex lg:max-w-lg lg:w-full md:w-1/2 w-5/6">
+            <div class="hidden md:flex absolute z-30 pt-20 m-14 w-48 h-52 xl:w-[400px] xl:h-[300px] bg-[#98A2B3] opacity-30 blur-2xl flex-none order-4 rounded-full"></div>
```
Comment (JayeshVP24) [inline src/components/Hero.astro]: - [x] add `pointer-events-none` in  class names

File: src/components/Navbar.jsx [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+                                <a
+                                    key={index}
+                                    className="text-[#E4E7EC] transition-all duration-500 hover:bg-[#475467] hover:text-[#E0EAFF] px-2 py-1 rounded-md text-md font-medium"
+                                    href={link.link}
+                                >
+                                    {link.title}
+                                </a>
+                            ))}
+
+                            <div className="ml-4 border-2 border-[#475467] rounded-lg">
```
Comment (JayeshVP24) [inline src/components/Navbar.jsx]: - [x] make ml-8

File: src/components/Navbar.jsx [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+                                <button className="inline-flex items-center text-[#E0EAFF] border-1 border-white py-1 px-2 focus:outline-none rounded md:mt-0 hover:underline text-base 2xl:text-lg"><a href='/Get Started'>Get Started</a>
```
Comment (JayeshVP24) [inline src/components/Navbar.jsx]: - [x] remove underline on hover and rather make a background color gray on hover

File: src/components/Navbar.jsx [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+        console.log(open);
+    };
+    return (
+        <div className='bg-[#101828] font-sans'>
+            <div className="mx-auto  px-4 sm:px-6 lg:px-8">
+                <div className="flex items-center justify-between h-16">
+                    {/* Logo */}
+                    <div className="flex items-center">
+                        <a href='/' className='text-white'>
+                            <img src="tcet-opensource-logo.png" alt="" srcSet="" />
```
Comment (JayeshVP24) [inline src/components/Navbar.jsx]: - [x] change src to svg logo

File: src/components/Hero.astro (hunk 2 above)
Comment (JayeshVP24) [reply src/components/Hero.astro]: - [x] remove `w-5/6`

Comment (JayeshVP24) [reply src/components/Hero.astro]: change `md:1/2` -> `sm:1/2`

File: src/components/Hero.astro [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+        
+          <div class="absolute z-30 py-0 left-0 w-80 h-96  bg-gradient-to-r from-teal-100 via-indigo-500 to-pink-100 opacity-30 blur-3xl flex-none order-4 rounded-xl transform -rotate-45"></div>
+
+          <div class="lg:flex-grow md:w-1/2 lg:pr-0 md:pr-12 flex flex-col md:items-start md:text-left mb-10 md:mb-0 items-center text-center font-title">
+            <h1 class="2xl:text-7xl xl:text-5xl sm:text-4xl text-3xl mb-2 text-white">Customise your computing experience with TCET Linux
+            </h1>
+            <p class="mb-8 leading-relaxed text-indigo-300 sm:text-sm text-xs">TCET Linux • Powered by FOSS</p>
+            <p class="mb-8 leading-relaxed text-[#D0D5DD] sm:text-base text-sm">TCET Open Source is an organization where software is not just a commodity to be bought and sold, but a collaborative effort that brings people together to solve problems and build something greater than themselves.</p>
+            <div class="flex justify-center">
+              <button class="inline-flex text-white bg-[#01105F] shadow-base shadow-sky-500 border-sky-500 border-y-2 py-2 px-6 focus:outline-none rounded-3xl text:base md:text-lg">Explore</button>
```
Comment (JayeshVP24) [inline src/components/Hero.astro]: - [x] add some hover and click effect

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]
Complete the given changes

**Review by riyavishwa981 — APPROVED** [review]

File: src/components/Navbar.jsx [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+                                <a
+                                    key={index}
+                                    className="text-[#E4E7EC] transition-all duration-500 hover:bg-[#475467] hover:text-[#E0EAFF] px-2 py-1 rounded-md text-md font-medium"
+                                    href={link.link}
+                                >
+                                    {link.title}
+                                </a>
+                            ))}
+
+                            <div className="ml-8 border-2 border-[#475467] rounded-lg">
```
Comment (JayeshVP24) [inline src/components/Navbar.jsx]: - [x] change `ml-8` to important as in `!ml-8`

File: src/components/Hero.astro [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+            <h1 class="2xl:text-6xl xl:text-5xl sm:text-4xl text-3xl mb-2 text-white">Customise your computing experience with TCET Linux
+            </h1>
+            <p class="mb-8 leading-relaxed text-indigo-300 sm:text-sm text-xs">TCET Linux • Powered by FOSS</p>
+            <p class="mb-8 leading-relaxed text-[#D0D5DD] sm:text-base text-sm">TCET Open Source is an organization where software is not just a commodity to be bought and sold, but a collaborative effort that brings people together to solve problems and build something greater than themselves.</p>
+            <div class="flex justify-center">
+              <button class="inline-flex text-white bg-[#01105F] shadow-base shadow-sky-500 border-sky-500 border-y-2 py-2 px-6 focus:outline-none rounded-3xl text:base md:text-lg hover:bg-blue-700 ">Explore</button>
+            </div>
+          </div> 
+
+          <div class=" flex lg:max-w-lg lg:w-full sm:w-1/2">
```
Comment (JayeshVP24) [inline src/components/Hero.astro]: - [x] remove `lg:max-w-lg lg:w-full `

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

**Review by JayeshVP24 — APPROVED** [review]
LGTM

**Pull request merged**